common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -3,8 +3,7 @@
   <div class="${status.classname}" id="status_${id}">
     <input type="text" name="input_${id}" id="input_${id}"
         data-input-id="${id}" value="${value|h}"
-        aria-label="${label}"
-        aria-describedby="${id}_status"
+        ${describedby}
```

Comment:
@muhammad-ammar Why do we need these changes and why we don't need `aria-label` ?